axlearn/cloud/gcp/scripts/project_setup.sh

```diff
@@ -0,0 +1,79 @@
+# Sets up a brand new GCP project for AXLearn. See also the "Getting Started" docs linked in the main readme.
+#
+# Usage: 
+#    # fill out environment variables below
+#    chmod +x project_setup.sh
+#    ./project_setup.sh
+
+# This will provision the following resources:
+#   * A new GCP project under your organization
+#   * A default VPC network
+#   * Three GCS buckets
+#   * A service account with the Storage Admin role
+
+# Note: Running this script does NOT grant you TPU quota.
+# Please follow instructions here: https://cloud.google.com/tpu/docs/setup-gcp-account#prepare-to-request 
+
+
+#!/bin/sh
+
+set -e
+
+# Define environment variables
+export ORGANIZATION_ID= #your existing GCP organization
+export BILLING_ACCOUNT_ID= #an existing billing account
+export PROJECT_ID= #the project name you want to create
+export NETWORK_NAME=default
+export TPU_REGION=
+export PERMANENT_BUCKET_NAME=${PROJECT_ID}-perm
+export PRIVATE_BUCKET_NAME=${PROJECT_ID}-private
+export TTL_BUCKET_NAME=${PROJECT_ID}-ttl
+export BUCKET_REGION=us-central1
+export SERVICE_ACCOUNT_NAME=my-service-account
+export SERVICE_ACCOUNT_ID=${SERVICE_ACCOUNT_NAME}@${PROJECT_ID}.iam.gserviceaccount.com 
+
+# Step 1: Create a GCP project
+# https://cloud.google.com/resource-manager/docs/creating-managing-projects
+gcloud projects create $PROJECT_ID --organization=$ORGANIZATION_ID
+gcloud config set project $PROJECT_ID
+
+# Step 2: Link the project to a billing account
+# https://cloud.google.com/billing/docs/how-to/modify-project
+gcloud billing projects link $PROJECT_ID --billing-account=$BILLING_ACCOUNT_ID
```

Comment:
I think we should assume the project has been created and a billing account has been linked. In most organizations this is a different team.